### ISSUE TITLE
Need to upgrade urllib3 to latest version 

### ISSUE THREAD

**Issue by SrividhyaGuru** [kennethreitz/requests#5085]
We are using piprot in our security build. recently our build is getting failed because of the following risk given by piprot.

```
python-piprot  medium  urllib3            Module is one or more minor versions out of date  Upgrade to v1.25.2 (Current: v1.24.2)
```

could you please let us know whether there is version upgrade possible in any time soon

Thanks in advance

**Comment by TheEdgeOfRage** [comment]
duplicate of #5067

**Comment by nateprewitt** [comment]
Resolving as duplicate. Issue is fixed in #5092.
